etc/helm/pachyderm/templates/pachd/identity-config.yaml

```diff
@@ -12,3 +12,18 @@ data:
     issuer: {{ include "pachyderm.issuerURI" . }}
     id_token_expiry: {{ .Values.oidc.IDTokenExpiry }}
     rotation_token_expiry: {{ .Values.oidc.RotationTokenExpiry }}
+  oidc-clients: |
+    - id: {{ .Values.pachd.oauthClientID }}
+      name: {{ .Values.pachd.oauthClientID }}
+      redirect_uris:
+      - {{ include "pachyderm.pachdRedirectURI" . }}
+      {{- if .Values.console.enabled}}
+      trusted_peers:
+      - {{ .Values.console.config.oauthClientID | quote }}
```

Comment:
Offtopic for this PR, but I'd love to be able to tack on my own trusted clients here.  For Hub, we had Jupyter on this list.  For our internal pachyderm instance, debug-dump-service is in here.